src/main/java/com/google/step/finscholar/data/College.java

```diff
@@ -0,0 +1,163 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.finscholar;
```

Comment:
Is the package supposed to be `com.google.step.finscholar.data`?

Reply:
No, in the pom.xml, the package is com.google.step.finscholar, the reason I'm using it is so that everything is in the same package and I can access classes within different folders in the java folder.

Reply:
You can still do that if you import them?